JitsiMeetJS.js

```diff
@@ -197,7 +197,11 @@ var LibJitsiMeet = {
               handler(error);
           });
         } else {
-            Statistics.sendUnhandledError(error);
+            if (error instanceof JitsiTrackError && error.gum) {
+                Statistics.sendGetUserMediaFailed(error);
+            } else {
+                Statistics.sendUnhandledError(error);
+            }
```

Comment:
Actually, I don't know why it's duplicated at all. I mean, why do we do the same in JitsiConference and in JitsiMeetJS.createLocalTracks. I'll ask in the chat probably.